structs.go

```diff
@@ -241,6 +241,72 @@ type Guild struct {
 	Unavailable                 bool              `json:"unavailable"`
 }
 
+// Role returns a role from the Guild.Roles slice with the given id
+func (g *Guild) Role(roleID string) *Role {
+	for _, r := range g.Roles {
+		if r.ID == roleID {
+			return r
+		}
+	}
+
+	return nil
+}
+
+// PresenceForUser returns a presence from the Guild.Presences slice with the given userID
+func (g *Guild) PresenceForUser(userID string) *Presence {
+	for _, p := range g.Presences {
+		if p.User.ID == userID {
+			return p
+		}
+	}
+
+	return nil
+}
+
+// VoiceState returns a voice state from the Guild.VoiceStates slice with the given userID
+func (g *Guild) VoiceState(userID string) *VoiceState {
+	for _, vs := range g.VoiceStates {
+		if vs.UserID == userID {
+			return vs
+		}
+	}
+
+	return nil
+}
+
+// Emoji returns a emoji from the Guild.Emojis slice with the given emojiID
+func (g *Guild) Emoji(emojiID string) *Emoji {
+	for _, e := range g.Emojis {
+		if e.ID == emojiID {
+			return e
+		}
+	}
+
+	return nil
+}
+
+// Member returns a member from the Guild.Members slice with the given userID
+func (g *Guild) Member(userID string) *Member {
+	for _, m := range g.Members {
+		if m.User.ID == userID {
+			return m
+		}
+	}
+
+	return nil
+}
+
+// Channel returns a channel from the Guild.Channels slice with the given channlID
```

Comment:
nit: you spelled channelID as channlID.

Super nit: All these comments should end in a period.